docs/usage/conventions.rst

```diff
@@ -5,7 +5,20 @@ This sections would introduce you conventions SCAutolib is using in sense of
 development, documentation, different approaches, and etc.
 
 1. Docstrings
-2. Packaging
-3. Unit test
-4. Architecture
-5. File structure
+----------------
+Each model, class, method, property or function has to have docstring.
+Description can be short, but has to be clear.
+For methods and function, docstring has have specification of arguments with data type, return value with data type and possible exceptions.
+
+2. Unit test
+--------------
+Test everything that can be tested.
```

Comment:
"everything" need to be better specified. Something like: All code should be covered by unit test unless there is specific reason preventing it.